src/common/lock.h

```diff
@@ -37,4 +37,7 @@ bool lockAcquire(
 // Release a lock
 bool lockRelease(bool failOnNoLock);
 
+// Write the backup percentage complete value into the backup lock file
+bool writeLockPercentageComplete( const String *lockPath, const String *stanza, const String *execId, const double percentagComplete);
+
```

Comment:
Remove extra space before const.

Reply:
And remove const before double. This const should only be in the C file -- the caller does not need to know we won't modify this param.

Reply:
updated in 0f56ed01..1f3fbd2b